library/src/morpheus-master/sass/modules/tool/resources/_resources.scss

```
@@ -259,37 +259,33 @@
 	}
 	
 	/* SAK-33897 - Restore + symbols in folders */
-	a.nil.fa-folder {
-		position: static;
+	a.nil.fa-folder, .nil a.fa-folder {
+		position: relative;
 		&::after {
 			color: var(--sakai-text-color-inverted);
 			content: "+";
 			display: inline-block;
+			font-family: monospace;
+			font-size: 11px;
 			position: absolute;
-			margin: 0.45em 0 0 -2.2em;
+			right: 4px;
```

Please use $standard-space:

```suggestion
			right: $standard-space;
```